src/template.js

```diff
@@ -305,6 +305,28 @@ export class Templates {
       resolver(templateClass);
     }
   }
+
+  /**
+   * @param {!BaseTemplate} impl
+   * @param {!JSONObject} data
+   * @private
+   */
+  render_(impl, data) {
+    const root = impl.render(data);
+    const anchors = root.getElementsByTagName('a');
+    for (let i = 0; i < anchors.length; i++) {
+      const anchor = anchors[i];
+      if (!anchor.hasAttribute('href')) {
+        // Ignore anchors without href.
+        continue;
+      }
+
+      // TODO(dvoytenko, #1572): This code should be unnecessary after
+      // sanitization issue has been addressed.
+      anchor.setAttribute('target', '_blank');
```

Comment:
This is ok since we process hash navigation separately in `document-click.js`. My hope, also, is to remove the whole thing once we resolve sanitizer problems.